src/main/java/org/opentripplanner/apis/transmodel/model/stop/StopTypeMapper.java

```diff
@@ -0,0 +1,26 @@
+package org.opentripplanner.apis.transmodel.model.stop;
+
+import graphql.schema.GraphQLEnumType;
+import org.opentripplanner.transit.model.site.StopType;
+
+/**
+ * Maps the StopType enum to a GraphQL enum.
+ */
+public class StopTypeMapper {
```

Comment:
I don't believe this is what @t2gran meant.

I think what he wanted was a simple function from enum to string.

What you have here is a type definition and, as weird as it sound, this class should be called `StopTypeType`.

But there are others that know the Transmodel API conventions better so I will let them weigh in.

Reply:
I believe he meant something like this: https://github.com/opentripplanner/OpenTripPlanner/blob/ffa411d8e9d5b60d4b72c312c9a8538a2015a158/src/main/java/org/opentripplanner/apis/transmodel/mapping/SeverityMapper.java#L14

Reply:
I like that mapper - but unsure if we can use the type in the api as is. Also, the name should be `StopTypeType` as @leonardehrenfried say.

Reply:
Changed to map to a string instead.

Reply:
Looks good!